Automattic/newspack-plugin
# feat(rss): add RSS Enhancements plugin to the core

### All Submissions:






### Changes proposed in this Pull Request:



Incorporates [`newspack-rss-enhancements` plugin](https://github.com/Automattic/newspack-rss-enhancements/) into the core plugin, as an optional module.

### How to test the changes in this Pull Request:

1. On a site with no `newspack-rss-enhancements` plugin installed, observe there's no "RSS Feeds" menu item
2. Visit Newspack Settings wizard by following a link in the plugin UI footer
3. Observe the module is turned off by default
4. Turn the module on, after navigating or refreshing the page observe the "RSS Feeds" menu item is available
5. Now on a site with `newspack-rss-enhancements` active, observe the module is turned on by default, but the `newspack-rss-enhancements` plugin has been deactivate on loading admin UI

### Other information:
